tests/suites/test_suite_psa_crypto_slot_management.function

```diff
@@ -62,6 +64,58 @@ static void psa_purge_key_storage( void )
 #define TEST_USES_KEY_ID( key_id ) ( (void) ( key_id ) )
 #endif /* MBEDTLS_PSA_CRYPTO_STORAGE_C */
 
+/** Apply \p close_method to invalidate the specified handles:
+ * close it, destroy it, or do nothing;
+ */
+static int invalidate_handle( close_method_t close_method,
+                              psa_key_handle_t handle )
+{
+    switch( close_method )
+    {
+        case CLOSE_BY_CLOSE:
```

Comment:
I'm not sure if we shouldn't mark intentional fallthroughs somehow, I can recall that some static analysis required that, but maybe not for MbedTLS.

Reply:
There's no fallthrough here, just two cases following each other. A fallthrough does require a comment. I recall the compiler settings we use on CI make it very picky about what this comment must contain.